config/application.rb

```diff
@@ -57,5 +61,21 @@ class Application < Rails::Application
     config.x.company_address      = ENV.fetch('COMPANY_ADDRESS', nil)
     config.x.company_website      = ENV.fetch('COMPANY_WEBSITE', nil)
     config.x.company_kvk          = ENV.fetch('COMPANY_KVK', nil)
+
+    config.x.site_name            = ENV.fetch('SITE_NAME', 'S.O.F.I.A.')
+    config.x.site_short_name      = ENV.fetch('SITE_SHORT_NAME', 'SOFIA')
+    config.x.site_association     = ENV.fetch('SITE_ASSOCIATION', 'C.S.V. Alpha')
+
+    config.x.code_beer            = ENV.fetch('CODE_BEER', nil)
+    config.x.code_craft_beer      = ENV.fetch('CODE_CRAFT_BEER', nil)
+    config.x.code_non_alcoholic   = ENV.fetch('CODE_NON_ALCOHOLIC', nil)
+    config.x.code_distilled       = ENV.fetch('CODE_DISTILLED', nil)
```

Comment:
some of these have default value `nil`. Why? if we want certain config variables to be sure to have a value, why not just throw an error, instead of providing the default values that belong to our own instance?